### PR TITLE
fix(php): improve the compatibility page

### PR DESCRIPTION
**Pull request by ghost**
Improvements to the PHP agent compatibility page.
Jira: NR-163392

### CODE REVIEW

**Comment by ally-sassman** [comment]
@ZNeumann Please review this PR and make any additional changes you see fit to improve this page.

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 1]
```diff
@@ -497,14 +472,13 @@ The PHP agent integrates with other New Relic features to give you end-to-end vi
...
       </td>
 
       <td>
-        New Relic CodeStream allows developers to see how the code they're responsible for is performing in production
-        by bringing observability into the IDE and making it part of their daily routine.
-        The PHP agent reports code-level metrics (starting with release 10.4.0) on your application which enables
-        detailed insight into how your code is performing at the method level.
+        New Relic CodeStream brings observability into the IDE to show how your code is performing in production at the method level.
+
+        The PHP agent versions 7.0 - 7.4 and 8.0 - 8.2 report code-level metrics (starting with release 10.4.0).
```
Comment (ZNeumann) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:481]:
```suggestion
        PHP versions 7.0 - 7.4 and 8.0 - 8.2 report code-level metrics (starting with release 10.4.0).
```
It is versions of PHP, not the agent. Though I am unsure how necessary this list is, as it encompasses (and will continue to encompass in the future) all PHP versions supported by the agent.

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 2]
```diff
@@ -13,67 +13,106 @@ signupBanner:
...
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                7.0 - 7.4
+            </td>
+            <td>
+                7.0 and 7.1 are for EOL in December 2023.
+            </td>
```
Comment (ZNeumann) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:41]:
```suggestion
```

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 3]
```diff
@@ -13,67 +13,106 @@ signupBanner:
...
+        </tr>
+    </tbody>
+</table> 
+
+## PHP agent compatibility
+
+See [PHP agent release notes](/docs/release-notes/agent-release-notes/php-release-notes/) for the latest PHP agent release.
+
+  * For PHP builds that are compiled with Zend Thread Safety (ZTS), only [PHP agent 9.19](/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309) and later supports these builds.
```
Comment (ZNeumann) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:60]:
```suggestion
  * For PHP builds that are compiled with Zend Thread Safety (ZTS), only [PHP agent 9.19](/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309) and earlier supports these builds.
```

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 4]
```diff
@@ -337,13 +354,13 @@ Supported databases and libraries:
   </tbody>
 </table>
 
-## Other APM software [#other-monitoring-software]
+## Other third-party monitoring software [#other-monitoring-software]
 
-If your application uses other application performance monitoring (APM) software besides our agent, we cannot guarantee that our agent will work correctly and we cannot offer technical support. For more information, see [Errors when using other monitoring software](/docs/apm/new-relic-apm/troubleshooting/errors-while-using-new-relic-apm-alongside-other-apm-software).
+If your application uses any additional monitoring software, we cannot guarantee that our agent will work correctly and cannot offer technical support. For more information, see [Errors when using other monitoring software](/docs/apm/new-relic-apm/troubleshooting/errors-while-using-new-relic-apm-alongside-other-apm-software).
 
 ## Instance details
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:361]: @ally-sassman, this looks like it's related to the main database section above. You might check with the SME to see if there's a way to combine these database instance details with the other database section. Or, consider at least moving this right below the database section.

**Review by rhetoric101 — DISMISSED** [review]
@ally-sassman nice revamp of this material! I just inserted a few minor suggestions for you to consider.

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 5]
```diff
@@ -13,67 +13,103 @@ signupBanner:
...
+* Ability to restart the web server or PHP process pool.
 
-## Permissions [#permissions]
+Running the agent **does not require** root access. The agent does need read/write access to the following files (the location of these files can be [customized in the PHP agent settings](/docs/php/php-agent-phpini-settings)):
 
-* Installation: Root access [is required](/docs/agents/php-agent/troubleshooting/determing-permissions-requirements) for most installations.
-* Running: Root access [is not required](/docs/agents/php-agent/troubleshooting/determing-permissions-requirements).
+* [A file storing the agent log files](/docs/php/php-agent-phpini-settings#inivar-daemon-logfile)
+* [A file storing the agent socket endpoint](/docs/php/php-agent-phpini-settings#inivar-daemon-port)
+* [A file storing the agent process ID](/docs/php/php-agent-phpini-settings#inivar-daemon-pidfile)
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:81]:
```suggestion
* [A file storing the agent log files](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#inivar-daemon-logfile)
* [A file storing the agent socket endpoint](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#inivar-daemon-port)
* [A file storing the agent process ID](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#inivar-daemon-pidfile)

```

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 6]
```diff
@@ -13,67 +13,105 @@ signupBanner:
...
+                Compatibility notes
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                7.0 - 7.4
+            </td>
+        </tr>
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:39]:
```suggestion
        <tr>
            <td>
                7.0 - 7.4
            </td>
                       <td>
                -
            </td>
        </tr>
```

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 7]
```diff
@@ -436,38 +418,30 @@ New Relic collects [instance details for a variety of databases and database dri
 
       <td>
         [redis](https://pecl.php.net/package/redis)
+        [predis](https://github.com/nrk/predis)
```
Comment (ally-sassman) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:421]:
```suggestion
        * [redis](https://pecl.php.net/package/redis)
        * [predis](https://github.com/nrk/predis)
```

**Comment by ally-sassman** [comment]
Made some changes after Rob's approval.

**Pull request merged**